account_due_list/views/payment_view.xml

```diff
@@ -9,25 +9,54 @@
         <field name="arch" type="xml">
             <tree
                 string="Payments"
-                decoration-muted="reconciled!=False"
+                decoration-success="full_reconcile_id!=False"
+                decoration-muted="reconciled!=False and full_reconcile_id==False"
                 decoration-danger="reconciled==False and date_maturity&lt;current_date"
                 create="false"
                 delete="false"
             >
-                <field name="move_id" string="Invoice" readonly="1" />
-                <field name="invoice_date" readonly="1" />
-                <field name="invoice_origin" readonly="1" />
-                <field name="name" readonly="1" />
-                <field name="partner_id" readonly="1" />
-                <field name="partner_ref" readonly="1" />
-                <field name="payment_term_id" readonly="1" />
+                <field name="date" readonly="1" optional="hide" />
+                <field name="invoice_date" readonly="1" optional="show" />
+                <field
+                    name="company_id"
+                    readonly="1"
+                    groups="base.group_multi_company"
+                    optional="hide"
+                />
+                <field name="journal_id" readonly="1" optional="hide" />
+                <field name="move_id" string="Invoice" readonly="1" optional="show" />
+                <field name="invoice_origin" readonly="1" optional="show" />
+                <field name="name" readonly="1" optional="show" />
+                <field name="partner_id" readonly="1" optional="show" />
+                <field name="partner_ref" readonly="1" optional="show" />
+                <field name="payment_term_id" readonly="1" optional="show" />
                 <field name="account_id" readonly="1" />
+                <field
+                    name="analytic_account_id"
+                    readonly="1"
+                    groups="account.group_account_user"
+                    optional="hide"
+                />
+                <field
+                    name="analytic_tag_ids"
+                    readonly="1"
+                    widget="many2many_tags"
+                    groups="analytic.group_analytic_tags"
+                    optional="hide"
+                />
                 <field name="debit" readonly="1" sum="Total Debit" />
```

Comment:
Not sure about these 2 being always shown. This is due list, not journal item view, so for me it's more important `balance`, `amount_currency` and `amount_residual` than these 2.